src/Chart.ts

```diff
@@ -40,22 +40,24 @@ export interface AnnotatedChartConfiguration extends ChartConfiguration {
   options: ChartOptionsWithAnnotation;
 }
 
-export function rescaleLabel(value: number, scale: number): string {
+export function rescaleLabel(value: string | number, scale: string | number): string {
```

Comment:
This change added as the build was failing with a typescript error (and failing in master too). I think the Chart.js types have updated to provide string | number (rather than number as previously) as the parameter for the callback which calls this method.